examples/java/pom.xml

```diff
@@ -194,6 +198,24 @@
           </execution>
         </executions>
       </plugin>
+
+      <!-- Override parent configuration.
+           Spark/Flink runner dependencies are necessary to be in shaded jar
+           in order to run WordCountIT with Spark/Flink runner -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.10</version>
```

Comment:
These versions are managed in the root `pom.xml` in the `pluginManagement` section. But in fact it is already configured to `ignoreNonCompile`, isn't it?